Description: Don't link extensions with the shared libpython library
 FIXME: Still needed since 3.8?

Bug-cpython: https://bugs.python.org/issue21536

--- a/Lib/distutils/command/build_ext.py
+++ b/Lib/distutils/command/build_ext.py
@@ -231,7 +231,7 @@ class build_ext(Command):
         # For building extensions with a shared Python library,
         # Python's library directory must be appended to library_dirs
         # See Issues: #1600860, #4366
-        if (sysconfig.get_config_var('Py_ENABLE_SHARED')):
+        if False and (sysconfig.get_config_var('Py_ENABLE_SHARED')):
             if not sysconfig.python_build:
                 # building third party extensions
                 self.library_dirs.append(sysconfig.get_config_var('LIBDIR'))
@@ -734,7 +734,7 @@ class build_ext(Command):
             # shared libraries are resolved at link time.
             from distutils.sysconfig import get_config_var
             link_libpython = False
-            if get_config_var('Py_ENABLE_SHARED'):
+            if False and get_config_var('Py_ENABLE_SHARED'):
                 # A native build on an Android device or on Cygwin
                 if hasattr(sys, 'getandroidapilevel'):
                     link_libpython = True
